src/config_files.rs

```diff
@@ -26,12 +26,25 @@ pub(crate) fn read_config_file(
     stack: &mut Stack,
     config_file: Option<Spanned<String>>,
     is_env_config: bool,
-    ask_to_create: bool,
+    load_defaults: bool,
+    create_scaffold: bool,
 ) {
-    warn!(
+    info!(
```

Comment:
The reason why all these were warn! is because log level warn is used for startup performance monitoring/checking, which reading & evaluating config files is a big part of. You can see this with `nu --log-level warn`. So, my preference is to change these all back from info! to warn!.

Reply:
@fdncred It's no problem to change them back, but can I double-check with you on that?  It seemed to me that developers who *aren't* working on startup performance probably didn't want that level of "noise" showing up when running with `--log-level warn`.

Can we filter `perf!` from `info!` with something like:

```nushell
ls /tmp/nu-*.log
| sort-by modified
| last
| get name
| open --raw $in
| lines
| each { parse -r '(.*?)\[ *(.*?) *\] (.*)'}
| flatten
| rename time level message
| update level {|log_line| if $log_line.message =~ ' perf:' {'PERF'} else {$log_line.level}}
| where level != PERF
```

I'll go without whatever you want here - I just wanted to double-check first.

Reply:
if we change all these to info! then it muddies the water, and you get a bunch of log entries that are not related to performance and it makes it more difficult to tell which entries are which. When I created this methodology ages ago you used to be able to do `nu --perf` and that would internally set the log level to info but people removed that. Now we just have info that is reserved for perf.

Technically, what I care about is showing _only_ performance information with some log-level. Right now, that's info, but we could change it to anything we want. Ideally it would be `--log-level perf` but i'm not sure there's a way to do that.

Reply:
Fair enough - I've reverted to `warn!`.  Thanks!